src/sgame/sg_cmds.cpp

```diff
@@ -631,8 +631,8 @@ static void Cmd_Give_f( gentity_t *ent )
 		if ( fabsf(amount) >= 1.0f )
 		{
 			valid = true;
```

Comment:
Is valid still required now?

Reply:
Yes, otherwise it prints an error message